.github/workflows/windows-vcpkg.yml

```diff
@@ -15,19 +15,19 @@ jobs:
     runs-on: windows-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     # Restore from cache the previously built ports. If "cache miss" then provision vcpkg,
     # install desired ports, finally cache everything for the next run.
     - name: Restore from cache and run vcpkg
-      uses: lukka/run-vcpkg@v5
+      uses: lukka/run-vcpkg@v10
       with:
-        vcpkgArguments: '--triplet x64-windows boost-asio boost-math boost-smart-ptr protobuf'
-        # Commit corresponding to vcpkg's master branch on 2020-11-20.
-        vcpkgGitCommitId: e803bf11296d8e7900dafb41e7b1224778d33dc6
+        #vcpkgArguments: '--triplet x64-windows'
+        # Commit corresponding to 2023.01.09
+        vcpkgGitCommitId: f14984af3738e69f197bf0e647a8dca12de92996
         # Workaround for https://github.com/ros-industrial/abb_libegm/pull/101#discussion_r433370614.
         # This line can be removed once protobuf is linked via its imported CMake targets.
-        vcpkgDirectory: '${{ github.workspace }}/../vcpkg'
+        #vcpkgDirectory: '${{ github.workspace }}/../vcpkg'
```

Comment:
This seems to still be necessary :(

Reply:
@traversaro: what's the current *modern cmake* way to get protobuf to behave? I notice you left a comment on this work-around.

If the old CMake version of older Ubuntu versions was what was holding us back, we might no longer have that restriction, so perhaps we can move to `protobuf::libprotobuf` et al. instead of `PROTOBUF_LIBRARIES`.